src/app/(social)/my-network/page.client.tsx

```
@@ -0,0 +1,202 @@
+
+              <FollowButton
+                onClick={() => {
+                  follow(user.id);
+                }}
+              />
+            </div>
+
+            <div
+              id="user-profile-description"
+              className="font-medium text-[12px] text-neutral-300"
+              dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)